src/search.rs

```diff
@@ -286,7 +286,7 @@ pub async fn search_install(config: &mut Config) -> Result<i32> {
         }
     }
 
-    let input = input(config, "Packages to install (eg: 1 2 3, 1-3 or ^4): ");
+    let input = input(config, "Packages to install (eg: 1 2 3, 1-3 or ^4):");
```

Comment:
The ^4 can go here too.

Reply:
You mean delete the "or ^4" from search ask input?

Reply:
yeah, I mean't "can go" as in "can be removed"

Reply:
Done. Please review it